src/Selenium2Library/keywords/formelement.py

```diff
@@ -1,51 +1,58 @@
 import os
 
-from .keywordgroup import KeywordGroup
+from Selenium2Library.base import Base
+from Selenium2Library.robotlibcore import keyword
+from Selenium2Library.keywords.element import ElementKeywords
 
 
-class FormElementKeywords(KeywordGroup):
+class FormElementKeywords(Base):
 
-    # Public, form
+    def __init__(self, ctx):
+        Base.__init__(self)
+        self.ctx = ctx
+        self.element_find = ElementKeywords(ctx).element_find
```

Comment:
This works, but moving the element finding logic into the context or into the base class would probably be cleaner.